pom.xml

```diff
@@ -42,27 +42,32 @@
             <artifactId>dp-dd-request-filter</artifactId>
             <version>v1.0.1</version>
         </dependency>
-        <dependency>
-            <groupId>com.github.ONSdigital</groupId>
-            <artifactId>dp-dd-backend-model</artifactId>
-            <version>1.0.30</version>
-        </dependency>
         <!--<dependency>-->
-            <!--<groupId>uk.co.onsdigital.discovery</groupId>-->
-            <!--<artifactId>dd-model</artifactId>-->
+            <!--<groupId>com.github.ONSdigital</groupId>-->
+            <!--<artifactId>dp-dd-backend-model</artifactId>-->
             <!--<version>1.0.30</version>-->
         <!--</dependency>-->
         <dependency>
+            <groupId>uk.co.onsdigital.discovery</groupId>
+            <artifactId>dd-model</artifactId>
+            <version>1.0.34</version>
+        </dependency>
+        <dependency>
```

Comment:
This probably needs to be reverted to use github version and updated to the one that you'll use after you merge your PR in the model.